src/config.js

```diff
@@ -33,8 +33,16 @@ module.exports = {
     hostname: process.env.ANALYTICS_HOSTNAME || "",
   },
 
+  postgres: {
+    host : process.env.POSTGRES_HOST,
+    user : process.env.POSTGRES_USER,
+    password : process.env.POSTGRES_PASSWORD,
+    database : process.env.POSTGRES_DATABASE || "analytics-reporter",
+  },
+
   static: {
-    path: '../analytics.usa.gov/'
+    path: '../analytics.usa.gov/',
   },
 
+  timezone: process.env.ANALYTICS_TIMEZONE || "US/Eastern",
```

Comment:
This should be transparent to users using it already. In order for the timezone to have any effect, they'll need to start using the `--write-to-database` option which until now has not been available.

That said, there is a concern that it could be applied to core functionality in the future without this consideration. With that in mind, how far should we reach to change the default here?